packages/client/src/intercom.js

```diff
@@ -0,0 +1,46 @@
+function intercom() {
+  (function () {
+    var w = window;
+    var ic = w.Intercom;
+    if (typeof ic === 'function') {
+      ic('reattach_activator');
+      ic('update', w.intercomSettings);
+    } else {
+      var d = document;
+      var i = function () {
+        i.c(arguments);
+      };
+      i.q = [];
+      i.c = function (args) {
+        i.q.push(args);
+      };
+      w.Intercom = i;
+      var l = function () {
+        var s = d.createElement('script');s
+          .type = 'text/javascript';
+        s.async = true;
+        s.src = 'https://widget.intercom.io/widget/gx67sx4f';
+        var x = d.getElementsByTagName('script')[0];
+        x.parentNode.insertBefore(s, x);
+      };
+      if (w.attachEvent) {
+        w.attachEvent('onload', l);
+      } else {
+        w.addEventListener('load', l, false);
+      }
+    }
+  })();
+
+  window.Intercom('boot', {
+    app_id: 'gx67sx4f',
+    name: 'Foretold', // Full name
+    email: 'internal-tech@foretold.io', // Email address
+    created_at: Math.floor(Date.now() / 1000),
```

Comment:
I think this should be the user's name/email/created at?

Reply:
Oh, sorry, sure, I did understand it from the first pass. Will change.